src/docfx/build/legacy/files/LegacyPage.cs

```diff
@@ -14,17 +14,9 @@ internal static class LegacyPage
             Document doc,
             LegacyManifestItem legacyManifestItem)
         {
-            // OPS build use TOC ouput as data page
```

Comment:
keep legacy TOC in this PR? legacy logic not added to BuildTableOfContents

Reply:
These lines are covered in LegacyTOC, I think they are kept here accidentally during refactor.

Reply:
impact test will tell us